assets/templates/subscriber-history.html

```diff
@@ -2795,6 +2795,13 @@ <h3>Estado GDPR del suscriptor</h3>
               </table>
             </div>
           </div>
+
+          <div class="dp-cta-links">
+            <a href="#"
+              ><span class="ms-icon icon-download"></span> DESCARGAR HISTORIAL
+              COMPLETO</a
```

Comment:
```suggestion
              ><span class="ms-icon icon-download"></span> Descargar historial
              completo</a
```

Reply:
the link has the uppercase, so it's the same

Reply:
Right, using uppercase is a style decision, so, the content should not be affected.